src/lib/mina_base/transaction_status.ml

```diff
@@ -37,6 +37,8 @@ module Failure = struct
         | Update_not_permitted_voting_for
         | Parties_replay_check_failed
         | Fee_payer_nonce_must_increase
+        | Account_precondition_unsatisfied
```

Comment:
We're going to want to make these more fine grained (but there are already issues for those, doesn't need to be here).